csrbuilder/__init__.py

```diff
@@ -5,9 +5,9 @@
 import re
 import sys
 import textwrap
+from functools import partial
 
 from asn1crypto import x509, keys, csr, pem
-from oscrypto import asymmetric
```

Comment:
We delay loading oscrypto until absolutely necessary.
  1. In case the public key is not asn1crypto.keys.PublicKeyInfo
  2. In case the private key does not conform to the duck typing